tasks/new_e2e_tests.py

```diff
@@ -177,6 +179,52 @@ def clean(ctx, locks=True, stacks=False):
     if stacks:
         _clean_stacks(ctx)
 
+    if output:
+        _clean_output()
+
+
+def _get_home_dir():
+    # TODO: Go os.UserHomeDir() uses a different algorithm than Python Path.home()
+    #       so a different directory may be returned in some cases.
+    return Path.home()
```

Comment:
so that TODO comment doesn't need to be at every usage of `Path.home()` and to make it easier to resolve the TODO in the future by replacing the content of one function instead of every usage of `Path.home()`.